docs/source/models/templates/concept_model_template.rst

```diff
@@ -72,188 +75,226 @@ Concept model template
 
 .. _{YOUR_MODEL_SHORT_NAME}1.0:
 
-1.0 Background
-++++++++++++++
-
-
-.. _{YOUR_MODEL_SHORT_NAME}1.1:
-
-1.1 Project overview
---------------------
-
-
-
-.. _{YOUR_MODEL_SHORT_NAME}1.2:
-
-1.2 Literature review
----------------------
-
-
-.. _{YOUR_MODEL_SHORT_NAME}2.0:
-
-2.0 Modeling aims and objectives
-++++++++++++++++++++++++++++++++
-
-
-.. _{YOUR_MODEL_SHORT_NAME}3.0:
-
-3.0 Causal framework
+1.0 Project overview
 ++++++++++++++++++++
+Here you can provide an introduction and overview to the project. Information
+in this section might include the following: 
 
-.. _{YOUR_MODEL_SHORT_NAME}3.1:
-
-3.1 Causal diagram
-------------------
- 
- .. note::
-    link to DAGs page
-    use round circles with DAGs
-
-**Outcome (O)**:
-
-
-
-**Most proximal determinant/exposure (E)**:
-  
-
-
-**Confounders (C)**:
-
-
-
-**Effect modifiers**:
-
-
-**Mediators (M)**:
-
-
-.. _{YOUR_MODEL_SHORT_NAME}3.2:
-
-3.2 Effect sizes
-----------------
-
-
-
-4.0 Intervention
-++++++++++++++++
-
+* Brief background of topic
+* Purpose or aims
+* Funder information
+* Examples of similar analyses
 
+.. _{YOUR_MODEL_SHORT_NAME}2.0:
 
-.. _{YOUR_MODEL_SHORT_NAME}4.1:
+2.0 Simulation Design
+++++++++++++++++++++++
 
-4.1 Simulation scenarios
-------------------------
+This section is intended as the main body of documentation of your simulation. Information
+in this section might include the following: 
 
+* Concept model diagram 
+* Links to relevant components of diagram
+* Scenario descriptions
+* Default specifications 
+* Demographics
+* Location description
+* Effect size
 
-.. _{YOUR_MODEL_SHORT_NAME}5.0:
+.. _{YOUR_MODEL_SHORT_NAME}2.1:
 
-5.0 Vivarium modelling components
-+++++++++++++++++++++++++++++++++
+2.1 Concept model diagram 
+-------------------------
 
-.. _{YOUR_MODEL_SHORT_NAME}5.1:
+.. _{YOUR_MODEL_SHORT_NAME}2.2:
 
-5.1 Vivarium concept model 
+2.2 Default specifications 
 --------------------------
 
-.. note::
-  This is our standard vivarium concept model diagram we are used to seeing
-
-.. _{YOUR_MODEL_SHORT_NAME}5.2:
-
-5.2 Demographics
-----------------
-
-.. _{YOUR_MODEL_SHORT_NAME}5.2.1:
+The below table is intended to outline the default specifications of your simulation. 
+Included in the table is a column of parameter definitions. Please delete this column as you 
+fill out the table. 
+
+.. list-table:: Default simulation specifications
+  :header-rows: 1
+
+  * - Parameter
+    - Definition
+    - Value
+    - Note
+  * - Location(s)
+    - Target/simulated region & GBD ID 
+    - e.g. Ethiopia (ID: 179)
+    -
+  * - Number of draws
+    - Desired number of draws that a given simulation is to be run for. (Generally, this should be a number between 0 and 999.)
```

Comment:
```suggestion
    - Desired number of draws that a given simulation is to be run for. (Generally, this should be a number between 1 and 999.)
```
I think we need at least one :)

Reply:
Good point! Lol (-:

Reply:
Wouldn't the maximum be 1000 then, not 999?